src/bootstraps.h

```diff
@@ -41,6 +41,24 @@ struct AltChainParamsVBTC : public altintegration::AltChainParams {
         return 0x3ae6ca;
     }
 
+    /** The maximum allowed PopData amounts per block */
+    uint32_t getMaxPopDataPerBlock() const noexcept override
+    {
+        return 50;
+    }
+
+    /** The maximum allowed weight for a PopData */
+    uint32_t getMaxPopDataWeight() const noexcept override
+    {
+        return 1000000;
+    }
+
+    uint32_t getSuperMaxPopDataWeight() const noexcept override
+    {
+        return 5 * getMaxPopDataWeight();
+    }
+
+
```

Comment:
I think these are no longer relevant?

We have a hard limit of 1 MB per block, we should use that for limits